govuk_sidekiq.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "gds-api-adapters", ">= 19.1.0"
   spec.add_dependency "govuk_app_config", ">= 1.1"
   spec.add_dependency "redis-namespace", "~> 1.6"
-  spec.add_dependency "sidekiq", ">= 6", "< 7"
```

Comment:
Ah, looks like this ended up in the wrong commit. Maybe this change could be made in the first commit?

Reply:
ah right, yep, see what you mean

Reply:
done